CHANGELOG.md

```diff
@@ -2,6 +2,26 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [4.0.0](https://github.com/awslabs/fhir-works-on-aws-deployment/compare/v3.1.0...v4.0.0) (2021-08-18)
+
+
+### ⚠ BREAKING CHANGES
+
+* The Cognito `IdToken` is now used instead of the `AccessToken` to authorize requests.
+
+Multi-tenancy itself is not a breaking change, you can continue to use FHIR works on single-tenant mode 
+by not using the `enableMultiTenancy` flag. 
+  
+However, note that updating an existing (single-tenant) stack to enable multi-tenancy is a breaking change. Multi-tenant 
+deployments use a different data partitioning strategy that renders the old, single-tenant, data inaccessible. 

```

Comment:
Do we have a script for this?

Reply:
There's no script to help make this migration.

Not every existing deployment out there benefits from switching to multi-tenancy. Some customers don't have multi-tenancy requirements and others prefer multiple separate (single-tenant) stacks for different tenants.

Migrating an existing stack from single-tenant to multi-tenant can be done by exporting all data (possibly with a single bulk export operation) and then inserting it into a new multi-tenant stack.

Reply:
Let's add this task to back log I can see this being a feature some customers would like/need